docs/api-gateway.md

```diff
@@ -207,26 +205,36 @@ Configuration of such API endpoint might look like this:
 
 > Note the presence of `response_from` field. This tells RIG to wait for different event with the same correlation ID.
 
-As an alternative you can set `response_from` to `http_async`. This means that correlated response has to be sent to internal `:4010/v2/responses` `POST` endpoint with a body like this:
+#### Supported combinations (`target` -> `response_from`)
+
+- HTTP -> `kafka`/`http_async`/`kinesis`
+- Kafka -> `kafka`
+- Kinesis -> **not supported**
+- Nats -> `nats`
+
+`http_async` means that correlated response has to be sent to internal `:4010/v2/responses` `POST` endpoint.
+
+#### Supported formats
+
+All `response_from` options support only binary mode.
```

Comment:
Not an issue, just a thought: is "binary mode" a common term outside the cloudevents spec?

Reply:
No idea tbh, but I guess it makes sense to be consistent with such terms.